src/com/content-item/vote-modal.js

```diff
@@ -0,0 +1,64 @@
+import { h, Component } 				from 'preact/preact';
+
+import SVGIcon 							from 'com/svg-icon/icon';
+
+import ButtonBase						from 'com/button-base/base';
+
+import ContentCommonBody				from 'com/content-common/common-body';
+
+import VoteResults						from 'com/content-item/vote-results';
+import VoteCurrent						from 'com/content-item/vote-current';
+import VoteVoting						from 'com/content-item/vote-voting';
+
+
+export default class VoteOrResults extends Component {
+	
+	constructor( props ) {
+		super(props);
+	}
+	
+	render(props, state) {
```

Comment:
Prefer `render( props, state ) {`, noticing the added spaces before the arguments. 

This is specifically in this case, because it defines a function. It's a subtle visual cue that this use of `blah( ... )` is defining a function instead of calling it.

Calling functions should have no prefix/suffix spaces (i.e. `render(props, state);`). 

`if` and `for` blocks should be treated similar to function definitions, but are sort of their own thing, as there should always be a space after a keyword (to help differentiate it from a function call). 

i.e. `if ( true ) {` or `for ( let idx = 0; idx < 10; idx++ ) {`.